erts/emulator/test/hipe_SUITE.erl

```diff
@@ -0,0 +1,67 @@
+%%
+%% %CopyrightBegin%
+%%
+%% Copyright Ericsson AB 2016. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% %CopyrightEnd%
+%%
+
+-module(hipe_SUITE).
+-export([all/0
+	,t_copy_literals/1
+	]).
+
+all() ->
+    case erlang:system_info(hipe_architecture) of
+	undefined -> {skip, "HiPE is disabled"};
+	_ -> [t_copy_literals
+	     ]
```

Comment:
Yes.  (I also find the current export ugly btw and would have liked it on one line too, but at least there I could perhaps see the reason.)